src/apis.ts

```diff
@@ -171,6 +171,11 @@ export const PUBLIC_ADVANCED_SERVICES: AdvancedService[] = [
     serviceId: 'analytics',
     version: 'v3',
   },
+  {
+    userSymbol: 'People',
+    serviceId: 'peopleapi',
```

Comment:
Are you sure "peopleapi" is correct? Looking at the Service Usage API, it seems like it should just be "people".

Reply:
You are right. When I use Service Usage API, it says serviceId is `people`.

However, If I clone the script with PeopleAPI enabled on the GUI, appscript.json says `serviceId:" peopleapi "`.

Reply:
Ah, you are right. This seems like another case of issue #512. It was assumed that the `serviceId` in  `appsscript.json` would be the same as the prefix used by the Service Usage API, but that isn't always the case. Until that issue is fixed we won't be able to support the People API fully.